src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -31,4 +31,6 @@ public class Messages {
             + String.format(MESSAGE_VIDEO_DOES_NOT_EXIST, "%2$s", "%3$s", "%4$s");
 
     public static final String MESSAGE_EMPTY_TAGS = "No tag is provided";
+
+    public static final String MESSAGE_2_CONFLICTING_ARGS = "The arguments %1$s and %2$s cannot both be specified";
```

Comment:
Can possibly remove the `_2_` unless theres a `MESSAGE_1_CONFLICTING_ARGS` variable?

Reply:
Thinking there might be a `MESSAGE_3_CONFLICTING_ARGS` or `MESSAGE_4_CONFLICTING_ARGS` in the future so wanted  to explicitly specify `MESSAGE_2_CONFLICTING_ARGS` so as to prevent unnecessary variable name changes. Also think it's more explicit this way.

Reply:
Im suggesting that we leave it as `MESSAGE_CONFLICTING_ARGS` for now and if there's a use for another conflicting arg variable, can insert it with `_2` then.

Reply:
alright, I'll commit the changes